src/Sulu/Bundle/MediaBundle/Resources/public/js/services/media-router.js

```diff
@@ -95,6 +96,14 @@ define([
                 }
             });
 
+            // show a single collection with files and upload (without locale)
+            routes.push({
+                route: mediaRoutes.editCollectionWithoutLocale,
+                callback: function(id) {
+                    MediaRouter.toCollection(id);
```

Comment:
Why is that new route necessary? And I would say it should be redirect to the URL with the locale. To what does the navigation link?